activesupport/lib/active_support/log_subscriber.rb

```diff
@@ -101,12 +108,29 @@ def flush_all!
         def fetch_public_methods(subscriber, inherit_all)
           subscriber.public_methods(inherit_all) - LogSubscriber.public_instance_methods(true)
         end
+
+        %w(info debug warn error fatal unknown).each do |level|
+          class_eval <<-METHOD, __FILE__, __LINE__ + 1
+            private def #{level}(method)
+              self.log_levels = log_levels.merge(method => ::Logger::#{level.upcase})
+            end
+          METHOD
+        end
```

Comment:
I found these new methods confusing on first read (particularly on the listeners which deal with two different log levels). How about something more explicit? maybe...

``` ruby
subscribe_log_level :enqueue_retry, :info
```

(non-blocking. If you prefer the current names stick with them!)